projects/etcd/java/README.md

```diff
@@ -0,0 +1,62 @@
+# java integration with etcd
+
+## Running etcd in docker
+
+```
+docker volume create --name etcd-data
+
+docker run \
+    -p 2379:2379 \
+    --restart=always \
+    --name etcd gcr.io/etcd-development/etcd:latest \
+    /usr/local/bin/etcd \
+    --data-dir=/etcd-data --name <hostname> \
+    --advertise-client-urls http://<host_ip>:2379 --listen-client-urls http://0.0.0.0:2379 \
+    --heartbeat-interval 1000 \
+    --election-timeout 5000
+```
+## java Integration
+Library Used - ``jetcd``
+
+### jetcd - Java Client for etcd
+jetcd is the official java client for etcd v3.
+
+### Java Versions
+  Java 8 or above is required.
+
+### Download
+  #### Maven
+```
+      <dependency>
+        <groupId>io.etcd</groupId>
+        <artifactId>jetcd-core</artifactId>
+        <version>${jetcd-version}</version>
+      </dependency>
+```
```

Comment:
1. Which nexus repo will one find this artifact.
2. what version was used in your example.

Reply:
Hi Rishi,

1. Maven Central will have this artifact.
    https://repo.maven.apache.org/maven2/io/etcd/jetcd-core/0.4.1/jetcd-core-0.4.1.jar
2. used version is 0.4.1